### PR TITLE
update rna transcription

### PR DESCRIPTION
**Pull request by zarniak**
checked with pep8
chekced with unit test
@virtuallight - dictionary solution :) do you approve?

### CODE REVIEW

File: rna_transcription.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
-def to_rna(dna_strand):
-    rna = ''
-    for x in dna_strand:
-        rna = rna + {
+dna_to_rna = {
```
Comment (virtuallight) [inline rna_transcription.py]: @zarniak 👍Minor improvement suggestion - usually, constants' names are all capital letters i.e. `DNA_TO_RNA`. That's to visually distinguish them from plain, mutable variables.

File: rna_transcription.py [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
             'T': 'A',
             'A': 'U'
-        }[x]
-    return rna
-print(to_rna('ACGTGGTCTTAA'))
+        }
+
+
+def to_rna(dna_strand):
+    return ''.join([dna_to_rna[x] for x in dna_strand])
```
Comment (virtuallight) [inline rna_transcription.py]: @zarniak 💯 

And... how would that look like with a built-in `map` function? :trollface:

Comment (zarniak) [reply rna_transcription.py]: @virtuallight like this :)
    return ''.join(list(map(dna_to_rna.get, dna_strand)))
dont know which is better - #1 is simpler to encode i think, #2 is simpler but need knowledge/expierence/google to encode

**Comment by virtuallight** [comment]
@zarniak Are the tests passing? :shipit:

**Pull request merged**